content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: An Alternative Istiod Deployment Model
```

Comment:
```suggestion
title: An External Control Plane Deployment Model
```

Reply:
"Deploying Istio control planes outside mesh clusters" ?

Much more clear and says what the post is about immediately

Reply:
"Deploying Istio Control Planes Outside the Mesh"